apis/telemetry/v1beta1/logpipeline_validation.go

```diff
@@ -1,20 +1,28 @@
 package v1beta1
 
 import (
+	"errors"
 	"fmt"
 	"regexp"
 	"strings"
 
 	"github.com/kyma-project/telemetry-manager/internal/fluentbit/config"
 )
 
+var (
+	validHostNameRE              = regexp.MustCompile(`^(([a-zA-Z0-9]|[a-zA-Z0-9][a-zA-Z0-9\-]*[a-zA-Z0-9])\.)*([A-Za-z0-9]|[A-Za-z0-9][A-Za-z0-9\-]*[A-Za-z0-9])$`)
```

Comment:
Please use the same name in v1alpha1 and v1beta1 validation for consistency.

Reply:
good catch